### PR TITLE
require platform-compatible phpunit version

### LINKED ISSUE #3320: Tests are broken on branch 2.3
See [this build](https://travis-ci.org/sonata-project/SonataAdminBundle/jobs/83453730#L129), for instance.

### PR DESCRIPTION
**Pull request by greg0ire**
Fixes #3320

### CODE REVIEW

**Comment by greg0ire** [comment]
Let's see if this works…

**Comment by greg0ire** [comment]
Nope :(

**Comment by greg0ire** [comment]
The idea does not work :(

**Comment by Soullivaneuh** [comment]
@greg0ire I have the same problem: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/pull/37

The only "solution" I have for the moment is to lock phpunit to version 4: https://github.com/sonata-project/SonataAdminBundle/commit/a2780f8788e03e1037f08fd5689119cb10849276

**Comment by Soullivaneuh** [comment]
Oh well, read too fast. My PR is not the same, I'm trying to add php platform composer config.

But this is no working either.

**Comment by greg0ire** [comment]
I'm trying to build something based on `version_compare` right now

**Comment by Soullivaneuh** [comment]
@greg0ire Why not just lock phpunit to 4 in waiting to find a better solution.

In any case, we can't use phpunit 5 API until we use phpunit 4.

**Comment by greg0ire** [comment]
until => 'as long as' ?

I think you right indeed, let's do this!

**Comment by greg0ire** [comment]
Good to merge!

File: .travis.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ before_script:
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
   - composer config -q -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - composer global require phpunit/phpunit fabpot/php-cs-fixer --no-update
+  - composer global require "phpunit/phpunit:^4.0" fabpot/php-cs-fixer --no-update
```
Comment (Soullivaneuh) [inline .travis.yml]: * No double quote
* `^4.8`

Will be easier to merge on master (modification already done).

**Comment by greg0ire** [comment]
Done

**Comment by Soullivaneuh** [comment]
Thank you @greg0ire.

**Pull request merged**